test/testlib/nuodb_database_utilities.go

```diff
@@ -189,8 +186,11 @@ func StartDatabaseTemplate(t *testing.T, namespaceName string, adminPod string,
 		})
 		// the TEs will become RUNNING after the SMs as they need an entry node
 		// so use the same ready timeout for both
-		readyTimeout := AdjustPodTimeout(tePodName, 300*time.Second)
-		AwaitPodUp(t, namespaceName, tePodName, readyTimeout)
+		readyTimeout := AdjustPodTimeout(tePodNameTemplate, 300*time.Second)
```

Comment:
Why don't we move this into the `if`-statement?

Reply:
The same variable is used to wait for the SMs in another _if_ block later (please see line 205).